src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -1 +1,101 @@
+package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** An item on a todo list. */
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/data")
+public class DataServlet extends HttpServlet {
+
+  private final List<Object> json = new ArrayList<>();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comments").addSort("score-class", SortDirection.ASCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    List<Entity> results =
+        datastore.prepare(query).asList(FetchOptions.Builder.withLimit(quantity));
+
+    for (Entity entity : results) {
+      String comment = (String) entity.getProperty("comments-class");
+      json.add(comment);
+    }
+
+    // Send JSON string.
+    String jsonVersion = new Gson().toJson(json);
+    response.setContentType("application/json;");
+    response.getWriter().println(jsonVersion);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get written feedback.
+    String classFeedback = request.getParameter("class-input");
+    String professorFeedback = request.getParameter("prof-input");
+    // Get class/professor ratings.
+    String ratingClass = request.getParameter("rating-class");
+    String ratingProfessor = request.getParameter("rating-professor");
+    String language = request.getParameter("languages");
+
+    Document doc =
+        Document.newBuilder().setContent(classFeedback).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float scoreClass = sentiment.getScore();
+    languageService.close();
+
+    Document doc =
```

Comment:
Not sure if this matters, but maybe name this docProfessor or something unqiue because doc is used above